### PR TITLE
fix: ldk use info log level by default

### PR DESCRIPTION
**Pull request by rolznz**
DEBUG can cause a lot of spam messages and the log files to get too large.

### CODE REVIEW

**Comment by rolznz** [comment]
Thanks @matjaz !

File: .env.example [hunk 1]
```diff
@@ -4,7 +4,7 @@ LOG_EVENTS=false
 # do not link your current account when you run a dev instance (so it stays pointing at your mainnet one)
 AUTO_LINK_ALBY_ACCOUNT=false
 
-# LDK trace log level
+# Optionally LDK debug log level to get more info
 #LDK_LOG_LEVEL=2
 # Logrus debug log level
 LOG_LEVEL=5
```
Comment (rolznz) [inline .env.example:10]:
```suggestion
LOG_LEVEL=4
```

**Comment by rolznz** [comment]
> also you should update this file https://getalby.com/install/hub/fly.toml

Edit: it's not set there, so it'll use the default value from Alby Hub which is fixed in this PR.

File: fly.toml [hunk 2]
```diff
@@ -12,7 +12,7 @@ swap_size_mb = 2048
 
 [env]
   DATABASE_URI = '/data/nwc.db'
-  LDK_LOG_LEVEL = '2'
+  LDK_LOG_LEVEL = '3'
   LOG_LEVEL = '5'
```
Comment (matjaz) [inline fly.toml:16]: this should be 4? same as in .env ?

Comment (rolznz) [reply fly.toml:16]: image

Info level is the one we want by default, which is 3

Comment (matjaz) [reply fly.toml:16]: in `.env.example` `LOG_LEVEL` is set to 4. in fly it's 5.

**Pull request merged**